Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-slim
+
+ARG FLEET_VERSION=compiler-fleet-1.0-SNAPSHOT
+
+ADD ./build/distributions/${FLEET_VERSION}.tar  .
+
+WORKDIR ./${FLEET_VERSION}/bin/
+
+ENTRYPOINT ["./compiler-fleet"]
```

Comment:
And you know, when `papeeria-compiler-fleet` is the default name of a project, the binary is also called `papeeria-compiler-fleet`.

Any sense to provide [distribution base name](https://docs.gradle.org/current/userguide/distribution_plugin.html) in build.gradle?

Reply:
Done